ci/workflows/pull-request.yml

```diff
@@ -155,7 +155,7 @@ steps:
           config: ci/docker-compose.yml
           mount-buildkite-agent: true
       - ./ci/plugins/upload-failure-logs
-    timeout_in_minutes: 7
+    timeout_in_minutes: 12
```

Comment:
I hit the timeout here. After discussing this with @fuyufjh, we decided to increase the timeout to 12 minutes.

Reply:
No need to increase timing, this PR should fix it: https://github.com/risingwavelabs/risingwave/pull/10452

Reply:
```suggestion
    timeout_in_minutes: 7
```